inbm/dispatcher-agent/dispatcher/aota/application_command.py

```diff
@@ -52,6 +52,38 @@ def __init__(self,
 
     def verify_command(self, cmd: str) -> None:
         check_application_command_supported(cmd)
+    
+    def run_signature_check(self, path_to_file: str) -> None:
```

Comment:
Consider refactoring the signature check logic into a separate method or class to improve modularity and testability. [medium]

Reply:
this is here on purpose as a utility function used by subclasses